src/mon/PGMap.cc

```diff
@@ -296,10 +296,10 @@ void PGMapDigest::print_oneline_summary(Formatter *f, ostream *out) const
   if (out)
     *out << num_pg << " pgs: "
          << states << "; "
-         << prettybyte_t(pg_sum.stats.sum.num_bytes) << " data, "
-         << kb_t(osd_sum.kb_used) << " used, "
-         << kb_t(osd_sum.kb_avail) << " / "
-         << kb_t(osd_sum.kb) << " avail";
+         << si_t(pg_sum.stats.sum.num_bytes) << " data, "
```

Comment:
i think the "B" in "KB" is missing here.

Reply:
yeah, that's true. My first version did not touch this field too, then it instead outputted:

`40 pgs: 40 active+clean; 785 MB data, 7.28G used, 22.9G / 30.2G avail; 453 kB/s rd, 59722 kB/s wr, 7244 op/s`

which was not also very consistent comparing to the origin version...